File: 706-Design-HashMap.py
class MyHashMap:
    def __init__(self):
        self.hashMap = {}

    def put(self, key: int, value: int) -> None:
        if key in self.hashMap:
            self.hashMap[key] = value
        else:
            self.hashMap[key] = value

    def get(self, key: int) -> int:
        if key in self.hashMap:
            return self.hashMap[key]
        return -1

    def remove(self, key: int) -> None:
        if key in self.hashMap:
            del self.hashMap[key]

File: neetcode/706-Design-HashMap.ts
class MyHashMap {
  private map: Map<number, number>

  constructor() {
    this.map = new Map<number, number>()
  }

  put(key: number, value: number): void {
    this.map.set(key, value)
  }

  get(key: number): number {
    const val = this.map.get(key)
    return val !== undefined ? val : -1
  }

  remove(key: number): void {
    this.map.delete(key)
  }
}